source/chapters/microphones.rst

```diff
@@ -430,9 +430,8 @@ The Strength knob adjusts the microphone talkover ducking strength and its behav
 Clicking  on the :guilabel:`Auto` button will switch it to any of three modes:
 
    * Off: When you speak through the microphone, the music  volume does not reduce.
-   * Auto: When you speak through the microphone, the music volume will be reduced when Mixxx detects your voice. You can adjust the amount by which the volume is automatically reduced with the Strength knob.
+   * Auto: When you speak through the microphone, the music volume will be reduced when Mixxx detects your voice. You can adjus4t the amount by which the volume is automatically reduced with the Strength knob.
```

Comment:
This introduces a typo.

Reply:
Ohhh my God. Let me get that fixed right now.

Reply:
hmmm....check still didn't pass..